woocommerce/woocommerce-android
# [Backend Receipts] Refactor current receipts implementation to be it payments wide

Closes: #10626


### Description

The PR doesn't bring any functional changes, just for sake of keeping PRs small, I move all related to receipts to the payments package as it will be not only card payments specific only

### Testing instructions

* Pay for the order with a card (or simulated card)
* Try print/view receipt
* Notice that there is no regression
